packages/mongodb/src/mongodb-store.service.ts

```diff
@@ -90,6 +90,54 @@ export class MongoDBStore extends SessionStore {
     });
   }
 
+  /**
+   * Returns all distinct session user id's.
+   */
+  async getAuthenticatedUserIds(): Promise<string[]> {
+    return await this.collection.distinct('state.userId', {
+      'state.userId': { $ne: null }
+    });
+  }
+
+  /**
+   * Destroys all sessions of a user.
+   *
+   * @param user
+   */
+  async destroyAllSessionsOf(user: { id: string }): Promise<void> {
+    await this.collection.deleteMany({ 'state.userId': user.id.toString() });
```

Comment:
I notice that if the ID of the user is `_id`, the functions won't work. Maybe we should change both functions of the MongoDB store and functions in TypeORM store in v3 to take the ID as parameter and not `user` objects.

Reply:
Yes that's correct, that's a little weird and needs a better solution. The problem lays in the missing ObjectID type, at least if it's dedicated to MongoDB and TypeORM. We could only take the id as a string as parameter which differs from the way Sessions are used with other databases in foal, but it would make it to be usable more universally across ORMs for MongoDB.

Reply:
After working with TypeORM v0.3 for Foal version 3, it appears that the ORM badly supports the use of `id` instead of `_id` for MongoDB with this version.

I think that we should then let the user use the form of their choice and thus take only the ID as parameter and not the user object. I updated the code of `TypeORMStore` accordingly for v3 to make the methods consistent across the framework (event though the ID type is different): #1152